server.js

```diff
@@ -3,35 +3,124 @@ import bodyParser from 'body-parser'
 import cors from 'cors'
 import mongoose from 'mongoose'
 
-// If you're using one of our datasets, uncomment the appropriate import below
-// to get started!
-// 
-// import goldenGlobesData from './data/golden-globes.json'
-// import avocadoSalesData from './data/avocado-sales.json'
-// import booksData from './data/books.json'
-// import netflixData from './data/netflix-titles.json'
-// import topMusicData from './data/top-music.json'
-
-const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/project-mongo"
+import netflixData from './data/netflix-titles.json'
+
+//Connecting to mongodb database
+const mongoUrl = process.env.MONGO_URL || "mongodb://localhost/NetflixItem"
 mongoose.connect(mongoUrl, { useNewUrlParser: true, useUnifiedTopology: true })
 mongoose.Promise = Promise
 
-// Defines the port the app will run on. Defaults to 8080, but can be 
-// overridden when starting the server. For example:
-//
-//   PORT=9000 npm start
+//Creating model of each instance that will be stored in database.
+const NetflixItem = mongoose.model('NetflixItem', {
+  title: String,
+  release_year: Number,
+  country: String,
+  show_id: Number,
+  type: String
+})
+
+//Seeding database and removing old items
+if (process.env.RESET_DB) {
+	const seedDatabase = async () => {
+    await NetflixItem.deleteMany({})
+
+		netflixData.forEach((item) => {
+			new NetflixItem(item).save()
+		})
+  }
+  seedDatabase()
+}
+
+// Defines the port the app will run on. 
 const port = process.env.PORT || 8080
 const app = express()
 
 // Add middlewares to enable cors and json body parsing
 app.use(cors())
 app.use(bodyParser.json())
 
-// Start defining your routes here
+//Handels 503 error e.g if database is unavilable
+app.use((req, res, next) => {
+  if(mongoose.connection.readyState === 1){
+    next()
+  } else {
+    res.status(503).json({ error: 'Service unavailable'})
+  }
+})
+
+//Displays first page 
 app.get('/', (req, res) => {
-  res.send('Hello world')
+  res.json('Welcome to Marjaanas Netflix API 🍿🍿🍿🍿  For documentation go to https://technigo-mongo-api.herokuapp.com/documentation');
```

Comment:
You could render documentation here, under the welcome text. It would safe your user one step to start using your service 😸